### ISSUE TITLE
Uncaught TypeError createElement is not a function - Webpack

### ISSUE THREAD

**Issue by rowellx68** [aurelia/dialog#295]
**I'm submitting a bug report**

* **Library Version:**
1.0.0-rc.1.0.3

**Please tell us about your environment:**
* **Operating System:**
OSX 10.12.5

* **Node Version:**
7.10.0

* **NPM Version:**
4.6.1

* **JSPM OR Webpack AND Version**
webpack 2.6.0

* **Browser:**
all

* **Language:**
all

**Current behavior:**
Throws a console error originating from [dialog-renderer](https://github.com/aurelia/dialog/blob/9a921230a34f8f487f8627e075e3f860ab9c5c72/src/dialog-renderer.ts#L33).

`Uncaught TypeError: __WEBPACK_IMPORTED_MODULE_0_aurelia_pal__.a.createElement is not a function`

The app works when running on webpack-dev-server, but not when it is compiled.

**Expected/desired behavior:**
Error should now happen on dev and build.

Steps to reproduce:
1. Clone the skeleton navigation project and choose either of the webpack projects
2. Add `aurelia-dialog` as dependency
3. Serve the app with `npm start` or `yarn start`

**Comment by StrahilKazlachev** [comment]
@PWKad If I remember correctly we've discussed this scenario already and decided that the dialog(or any other module depending on PAL) should not be loaded before the PAL is initialized, right?

**Comment by rowellx68** [comment]
Is there a possible workaround for this bug?

**Comment by PWKad** [comment]
@StrahilKazlachev seems correct I thought those in the know with WebPack mentioned how this should not be able to happen (I thought) but if it is not working when compiled does that just mean that you forgot to include polyfills?  Not very familiar with webpack...

**Comment by StrahilKazlachev** [comment]
@rowellx68 could you see if this helps - aurelia/webpack-plugin#103?

**Comment by doubleaa93** [comment]
I'm getting same error but not when compiled but when running karma and testing a component that imports the DialogService

**Comment by StrahilKazlachev** [comment]
Closing since I think aurelia/webpack-plugin#103 explains quite well the issue and what to do.

**Comment by rowellx68** [comment]
@StrahilKazlachev 

A few things seems to have solved my problem.

>Many users have taken the (bad) habit of listing all their aurelia modules as entry point in a vendor chunk and then including that in the page.
>
>https://github.com/aurelia/pal-browser/issues/17#issuecomment-290951116

And then I was getting an error with the SVGAnalyzer which was solved with this workaround.
https://github.com/aurelia/validation/issues/445#issuecomment-305650958

**Comment by jmzagorski** [comment]
@doubleaa93 did you get the issue with karma resolved? I experienced the same issue using `dotnet new aurelia` and adding just the necessary packages to get karma + jasmine working. I created one dummy test and it failed as soon as i imported `DialogService`, receiving the error `__WEBPACK_IMPORTED_MODULE_0_aurelia_pal__.DOM.createElement is not a function`

**Comment by doubleaa93** [comment]
@jmzagorski I got it to work by adding this to the top of karma-bundle.js

```
import 'aurelia-polyfills';
import 'aurelia-loader-webpack';
import {initialize} from 'aurelia-pal-browser'

initialize();
```

**Comment by bulldetektor** [comment]
Any idea on when this will be shipped? Latest release on NPM is 1.0.0-rc.1.0.3 from May 5th.

**Comment by StrahilKazlachev** [comment]
@bulldetektor All is ready for release, waiting @EisenbergEffect to release it.
